test/integration/docker-app/wdio.conf.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
```

Comment:
I think WDIO5 changed the you pass arguments into Chrome - this config uses flags to make chrome run headless, so it will need to change. I believe new way is 
```js
'goog:chromeOptions': {
        args: ['--headless', '--disable-gpu'],
}
```

Reply:
Agree